core/src/main/java/nl/nn/adapterframework/filesystem/AmazonS3FileSystem.java

```diff
@@ -603,5 +608,19 @@ public void setBucketCreationEnabled(boolean bucketCreationEnabled) {
 	public boolean isBucketExistsThrowException() {
 		return bucketExistsThrowException;
 	}
+
+	public ClientConfiguration getProxyConfig() {
+		return proxyConfig;
+	}
+
+	public void setProxyConfig(String proxyHost, Integer proxyPort) {
+		if (proxyHost != null && proxyPort != null) {
+			proxyConfig = new ClientConfiguration();
+			proxyConfig.setProtocol(Protocol.HTTPS);
+			proxyConfig.setProxyHost(proxyHost);
+			proxyConfig.setProxyPort(proxyPort);
+		}
+	}
```

Comment:
You'll have to create two separate setters for this: one for proxyHost and one for proxyPort.
Creating these two setters results in the possibility to set values in the configuration.xml, like:
proxyHost="hq.proxy.insim.biz" proxyPort="9400"

Reply:
ok, i will change it